src/assets.rs

```diff
@@ -25,22 +25,23 @@ pub(super) fn plugin(app: &mut App) {
 pub struct ImageHandles(HashMap<String, Handle<Image>>);
 
 impl ImageHandles {
-    pub const KEY_DUCKY: &'static str = "Ducky";
+    pub const KEY_DUCKY: &'static str = "images/ducky.png";
 }
 
 impl FromWorld for ImageHandles {
     fn from_world(world: &mut World) -> Self {
         let asset_server = world.resource::<AssetServer>();
-        let map = [(
-            ImageHandles::KEY_DUCKY.to_string(),
-            asset_server.load_with_settings(
-                "images/ducky.png",
-                |settings: &mut ImageLoaderSettings| {
-                    settings.sampler = ImageSampler::nearest();
-                },
-            ),
-        )]
-        .into();
+
+        let pixel_art_settings = |settings: &mut ImageLoaderSettings| {
+            // Using nearest sampling to preserve pixel art style
```

Comment:
```suggestion
            // Use `nearest` image sampling to preserve the pixel art style.
```

Reply:
Or we could mention `ImageFilterMode::Nearest`? That's probably overkill though.